pkg/nodes/transform/transform.go

```
@@ -21,9 +24,35 @@ func (transformDefinition) Meta() api.NodeType {
 
 // ExecuteEnvelope applies the expression to the input envelope (currently passthrough).
 func (d transformDefinition) ExecuteEnvelope(ctx api.ExecutionContext, node api.Node, envelope *api.Envelope[interface{}]) (*api.Envelope[interface{}], error) {
-	// TODO: Implement actual transformation logic based on expression
+	val, ok := node.Data["expression"]
```

golangci-lint (1.64.8)

ineffectual assignment to ok

(ineffassign)